.github/workflows/test-module.yml

```diff
@@ -0,0 +1,64 @@
+name: Test a module with CITGM
+
+on:
+  workflow_dispatch:
+    inputs:
+      module:
+        description: 'Module to test'
+        required: true
+      repository:
+        description: 'CITGM repository to checkout'
+        required: true
+        default: nodejs/citgm
+      ref:
+        description: 'CITGM GitHub ref to checkout'
+        required: true
+        default: main
+
+jobs:
+  show-parameters:
+    name: Parameters for testing "${{ github.event.inputs.module }}"
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Log workflow parameters
+        env:
+          WORKFLOW_PARAMETERS: ${{ toJSON(github.event.inputs) }}
+        run: echo "$WORKFLOW_PARAMETERS"
+
+  test-module:
+    name: Test on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
```

Comment:
if you omit this name, then the list should automatically show the matrix parameters in parens